scripts/basic-workflow-tests.sh

```diff
@@ -134,12 +134,16 @@ test_train() {
     task Train the model
 
     # TODO Only cuda for now
-    TRAIN_ARGS="--device=cuda --4-bit-quant"
+    TRAIN_ARGS=()
+    TRAIN_ARGS+=("--device=cuda")
+    TRAIN_ARGS+=("--override-training-args")
+    TRAIN_ARGS+=("{\"bf16\":false, \"gradient_checkpointing\":true, \"gradient_accumulation_steps\":8}")
```

Comment:
The e2e job failed here

https://github.com/instructlab/instructlab/actions/runs/8947148148/job/24578720173#step:10:13

```
./scripts/basic-workflow-tests.sh: 137: Syntax error: "(" unexpected (expecting "}")
```

Reply:
probably because it uses `/bin/sh` instead of `/bin/bash` ?

Reply:
I made the change to `/bin/bash` and kicked off another run -- https://github.com/instructlab/instructlab/actions/runs/8947293535